### PR TITLE
add missing type

### PR DESCRIPTION
**Pull request by fsx950223**

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
can you run `bash tools/pre-commit.sh`? Thanks!

File: tensorflow_addons/losses/quantiles.py [hunk 1]
```diff
@@ -118,14 +119,14 @@ class PinballLoss(tf.keras.losses.Loss):
       - https://en.wikipedia.org/wiki/Quantile_regression
       - https://projecteuclid.org/download/pdfview_1/euclid.bj/1297173840
     """
-
+    @typechecked
```
Comment (autoih) [inline tensorflow_addons/losses/quantiles.py]: Thanks @fsx950223. Here may need a blank line.

File: tensorflow_addons/losses/quantiles.py [hunk 2]
```diff
@@ -118,14 +119,14 @@ class PinballLoss(tf.keras.losses.Loss):
...
     def __init__(
-        self, tau=0.5, reduction=tf.keras.losses.Reduction.AUTO, name="pinball_loss"
+        self, tau: float=0.5, reduction: tf.keras.losses.Reduction=tf.keras.losses.Reduction.AUTO, name: str="pinball_loss"
```
Comment (autoih) [inline tensorflow_addons/losses/quantiles.py]: tau: FloatTensorLike = 0.5

**Comment by fsx950223** [comment]
@autoih @gabrieldemarmiesse Fixed.

File: tensorflow_addons/losses/quantiles.py [hunk 3]
```diff
@@ -119,13 +123,17 @@ class PinballLoss(tf.keras.losses.Loss):
       - https://projecteuclid.org/download/pdfview_1/euclid.bj/1297173840
     """
 
+    @typechecked
     def __init__(
-        self, tau=0.5, reduction=tf.keras.losses.Reduction.AUTO, name="pinball_loss"
+        self,
+        tau: FloatTensorLike = 0.5,
+        reduction: tf.keras.losses.Reduction = tf.keras.losses.Reduction.AUTO,
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/losses/quantiles.py]:
```suggestion
        reduction: str = tf.keras.losses.Reduction.AUTO,
```
```
>>>isinstance(tf.keras.losses.Reduction.AUTO, tf.keras.losses.Reduction)
False
>>> type(tf.keras.losses.Reduction.AUTO)
str
```

**Comment by gabrieldemarmiesse** [comment]
Thanks!

**Pull request merged**